### PR TITLE
Remove useAPI from DeleteUser and DeleteIDPConfigurationModal

### PR DESCRIPTION
**Pull request by dvaldivia**

### CODE REVIEW

**Comment by cesnietor** [comment]
@dvaldivia :
```
uilding Console binary to './console'
# github.com/minio/console/api/operations/user
api/operations/user/check_user_service_accounts_parameters.go:58:23: undefined: models.SelectedUsers
api/operations/user/check_user_service_accounts_parameters.go:72:19: undefined: models.SelectedUsers
make: *** [console] Error 1
```

missing file models/selected_users.go

File: web-app/src/screens/Console/IDP/DeleteIDPConfigurationModal.tsx [hunk 1]
```diff
@@ -39,22 +40,27 @@ const DeleteIDPConfigurationModal = ({
...
 
   if (!idp) {
     return null;
   }
 
   const onConfirmDelete = () => {
-    invokeDeleteApi("DELETE", `/api/v1/idp/${idpType}/${idp}`);
+    setDeleteLoading(true);
+    api.idp
+      .deleteConfiguration(idpType, idp)
```
[the remark this answers is not in the document]
Comment (dvaldivia) [reply web-app/src/screens/Console/IDP/DeleteIDPConfigurationModal.tsx:58]: Fixed

[the remark this answers is not in the document]
Comment (cesnietor) [reply web-app/src/screens/Console/IDP/DeleteIDPConfigurationModal.tsx:58]: push it :P

[the remark this answers is not in the document]
Comment (dvaldivia) [reply web-app/src/screens/Console/IDP/DeleteIDPConfigurationModal.tsx]: hahaha totally missed that part

**Review by reivaj05 — APPROVED** [review]

**Review by cesnietor — APPROVED** [review]

**Pull request merged**